app/controllers/api/v1/groups/statuses_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Api::V1::Groups::StatusesController < Api::BaseController
+  include Authorization
+  include Payloadable
+
+  before_action -> { authorize_if_got_token! :write, :'write:groups' }
+  before_action :set_status
+
+  def destroy
+    authorize @status.group, :delete_posts?
+    #TODO: refactor
+    #TODO: logging
+    #TODO: notifying user
+    if @status.local?
+      RejectGroupStatusService.new.call(@status) #TODO: federation
```

Comment:
Note for later

Reply:
Not sure why I kept this comment as I implemented sending a `Remove` as appropriate. Will remove this comment.